website-frontend/src/styles/bootstrap/index.scss

```diff
@@ -1,2 +1,6 @@
 @import './overrides.scss';
-@import 'bootstrap/scss/bootstrap.scss';
\ No newline at end of file
+@import 'bootstrap/scss/bootstrap.scss';
+
+#hero {
+  height: calc(100vh - #{$spacer} * 2 - #{$nav-link-height} - #{$nav-link-padding-y * 2});
+}
```

Comment:
`#hero` does not belong to `bootstrap`, so it should not be in `bootstrap/` em. To make this better, please create `styles/app_variables.scss` or something like that, move this code there, and import it into `styles/index.scss`.

Reply:
Btw, does this expression work?

Reply:
I have forgotten about so, thank you for adding on anh ơi. And I have added that also.
Fyi, the expression works ạ.